packages/core/src/lib/connection_manager.ts

```diff
@@ -0,0 +1,360 @@
+import { Connection } from "@libp2p/interface-connection";
+import { PeerId } from "@libp2p/interface-peer-id";
+import { PeerInfo } from "@libp2p/interface-peer-info";
+import { Peer } from "@libp2p/interface-peer-store";
+import { IRelay, Tags } from "@waku/interfaces";
+import debug from "debug";
+import type { Libp2p } from "libp2p";
+
+import KeepAliveManager, { KeepAliveOptions } from "./keep_alive_manager.js";
+
+const DEFAULT_MAX_BOOTSTRAP_PEERS_ALLOWED = 2;
```

Comment:
```suggestion
const DEFAULT_MAX_BOOTSTRAP_PEERS_ALLOWED = 1;
```
Just one for now, the bootstrap fleet is small.